linux-packages/content/configure-tentacle.sh

```diff
@@ -149,6 +150,25 @@ function setupPollingTentacle {
     read -p "What name would you like to register this Tentacle with? ($displayname): " displaynameinput
     displayname=$(assignNonEmptyValue "$displaynameinput" $displayname)
 
+    $doesconnecttosamedomain=1
+    read -p 'Is the comms port on the same domain as the Octopus Server : 1) Yes or 2) No (default 1): ' doesconnecttosamedomain
+
+    
+    
+    case $doesconnecttosamedomain in
+        2)
+            read -p "What is the Octopus Server comms address including port e.g. 'https://polling.<yoururl>.octopus.app:443' ?: " commsAddress
```

Comment:
[The public doco](https://octopus.com/docs/infrastructure/deployment-targets/tentacle/polling-tentacles-over-port-443) doesn't mention that the port number needs to be specified. Would it work if we do?

Reply:
I requested it since it makes the bash easier, otherwise we then need to ask for port if a port is not set.

Also yes this does work.